django/thunderstore/api/cyberstorm/views/package_detail.py

```diff
@@ -0,0 +1,152 @@
+from django.db.models import CharField, Count, OuterRef, QuerySet, Subquery, Sum, Value
+from rest_framework import serializers
+from rest_framework.generics import RetrieveAPIView, get_object_or_404
+
+from thunderstore.api.cyberstorm.serializers import (
+    CyberstormPackageCategorySerializer,
+    CyberstormTeamMemberSerializer,
+)
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.community.models.package_listing import PackageListing
+from thunderstore.repository.models.package import get_package_dependants
+from thunderstore.repository.models.package_version import PackageVersion
+
+
+class DependencySerializer(serializers.Serializer):
+    """
+    Dependencies of a given PackageVersion, listed in a given Community.
+
+    community_identifier and namespace are not present by default and
+    need to be annotated to the object.
+    """
+
+    community_identifier = serializers.CharField()
+    description = serializers.CharField()
+    icon_url = serializers.CharField(source="icon.url")
+    name = serializers.CharField()
+    namespace = serializers.CharField(source="package.namespace.name")
+    version_number = serializers.CharField()
+
+
+class TeamSerializer(serializers.Serializer):
+    """
+    Minimal information to present the team on package detail view.
+    """
+
+    name = serializers.CharField()
+    members = CyberstormTeamMemberSerializer(many=True)
```

Comment:
The comment about truncation probably applies here too, but less likely to be a problem in practice than dependencies I feel like.